server/nlp/nlp.py

```diff
@@ -58,6 +59,11 @@ def set_similarity_data(requirements, topic_summaries):
         )
         NLProcessor.__sim_topic_summaries = topic_summaries
 
```

Comment:
missing method comment

Reply:
removed the method as it belongs to our previous censorship model and is no longer used